js-exercises/object-utils/objectUtils.js

```diff
@@ -0,0 +1,28 @@
+function getKeyDescriptorPairs(object) {
+  return Object.getOwnPropertyNames(object).map((key) => {
+    const desc = Object.getOwnPropertyDescriptors(object, key);
+    return [key, desc];
+  });
+}
+
+function fromKeyDescPairs(pairs) {
+  return pairs.reduce((result, [key, desc]) => {
+    Object.defineProperties(result, key, desc);
+    return result;
+  }, {});
```

Comment:
Again, use of reduce function is used as a for loop and is wrong. Instead a for loop would have sufficed